fmfexporter/connectors/jira/fmf_jira.py

```diff
@@ -0,0 +1,108 @@
+import configparser
+import jira
+
+from fmfexporter.adapters import PolarionTestCase
+
+
+class JiraConfig(object):
+    """
+   PolarionConfig represents data that must be provided through
+   config (ini) file (to enable communication with the polarion importer APIs)
+   """
+
+    KEY_SECTION = 'jira'
+    KEY_PROJECT = 'project'
+    KEY_URL = 'url'
+    KEY_USERNAME = 'username'
+    KEY_PASSWORD = 'password'
+
+    def __init__(self, config_file):
+        self.config = configparser.ConfigParser()
+        self.config.read(config_file)
+        assert JiraConfig.KEY_SECTION in self.config.sections()
+
+    @property
+    def project(self) -> str:
+        """
+        Returns the parsed jira project name
+        :return:
+        """
+        return self.config[JiraConfig.KEY_SECTION][JiraConfig.KEY_PROJECT]
+
+    @property
+    def url(self) -> str:
+        """
+        Returns the parsed jira project url
+        :return:
+        """
+        return self.config[JiraConfig.KEY_SECTION][JiraConfig.KEY_URL]
+
+    @property
+    def username(self) -> str:
+        """
+        Returns the parsed jira username
+        :return:
+        """
+        return self.config[JiraConfig.KEY_SECTION][JiraConfig.KEY_USERNAME]
+
+    @property
+    def password(self) -> str:
+        """
+        Returns the parsed jira password
+        :return:
+        """
+        return self.config[JiraConfig.KEY_SECTION][JiraConfig.KEY_PASSWORD]
+
+
+class FMFJiraPopulator(object):
+    TEST_WI = 'test-work-item'
+    QE_TEST_COV = 'qe-test-coverage'
+    VERIFIED_IN_REL = 'verified-in-release'
+
+    def __init__(self, config_file):
+        self.config = JiraConfig(config_file)
+        credentials = (self.config.username, self.config.password)
+        self.jira_login = jira.JIRA(self.config.url,
+                              basic_auth=credentials)
+        if self.config.project.lower() == "entmqbr":
+            self.custom_fields = FMFJiraPopulator.get_entmqbr_custom_fields()
+
+    def populate_testcases(self, tc_list):
```

Comment:
What is the tc_list? List of Polarion TC right? Can we get explicit typing?

Reply:
Actually it can be *any* TestCase, which has defects filled in. 
Sadly I have no idea to tell, which TestCase class is has to be, as PolarionTestCase inherits from object :/

Reply:
Done.